apps/nextjs-website/src/helpers/getIconFromName.tsx

```diff
@@ -0,0 +1,19 @@
+import { HeadsetMic } from '@mui/icons-material';
+
+export type IconName = 'HeadsetMic';
```

Comment:
You should also handle the `FeedbackIcon`.
You can import it with `import FeedbackIcon from '@mui/icons-material/Feedback';`